comms/lighthouse/src/routes.ts

```diff
@@ -32,14 +32,15 @@ export function configureRoutes(
 ) {
   const { configService } = services
 
-  const getStatus: RequestHandler = (req, res) => {
+  const getStatus = async (req: Request, res: Response) => {
     const status: any = {
       name: options.name,
       version: options.version,
       currenTime: Date.now(),
       env: options.env,
       ready: true,
       usersCount: services.peersService().getActivePeersCount(),
+      islandsCount: await services.archipelagoService().getIslandsCount(),
```

Comment:
is `getIslandsCount()` cached?

Reply:
Is in memory. The only overhead is inter process communication, though it should be not much. Likely equivalent to requesting a key from a Redis running in the same machine or something similar